src/vsg/utils/PolytopeIntersector.cpp

```diff
@@ -78,7 +78,7 @@ namespace vsg
             {
                 size_t numNegativeDistances = 0;
                 size_t numPositiveDistances = 0;
-                size_t numZeroDistances = 0;
+                [[maybe_unused]] size_t numZeroDistances = 0;
```

Comment:
Suppressing the warning with an attribute is still a bad idea. Either we figure out what the variable was intended to be used for and use it for that, or we get rid of it.